pkg/vm/stackitem/item.go

```diff
@@ -60,6 +60,13 @@ type Convertible interface {
 	FromStackItem(Item) error
 }
 
+// Equatable describes a special value of Interop that can be compared with
+// value of some other InteropInterface.
+type Equatable interface {
+	// Equals checks if two objects are equal.
+	Equals(other any) bool
```

Comment:
`other Equatable` just to avoid `any` and associated interface conversion?

Reply:
Nice idea, fixed.